src/main/java/seedu/address/logic/commands/AddPatientCommand.java

```diff
@@ -0,0 +1,80 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DOB;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NRIC;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Patient;
+
+/**
+ * Adds a person to the address book.
+ */
+public class AddPatientCommand extends Command {
+
+    public static final String COMMAND_WORD = "addpatient";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a patient to the address book. "
+            + "Parameters: "
+            + PREFIX_NRIC + "NRIC "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_DOB + "DOB "
+            + PREFIX_PHONE + "PHONE "
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NRIC + "S1234567A "
+            + PREFIX_NAME + "John Doe "
+            + PREFIX_DOB + "2003-01-30 "
+            + PREFIX_PHONE + "98765432";
+
+    public static final String MESSAGE_SUCCESS = "New patient added: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This patient already exists in the address book";
```

Comment:
I think we change from "exists in address book" to "exists in patient database" or similar.